web/components/GridWrapper/GridWrapper.tsx

```diff
@@ -0,0 +1,10 @@
+import { ReactNode } from 'react';
+import styles from './GridWrapper.module.css';
+
+interface GridWrapperProps {
+  children: ReactNode;
+}
+
+export const GridWrapper = ({ children }) => (
```

Comment:
```suggestion
export const GridWrapper = ({ children }: GridWrapperProps) => (
```

Reply:
Thanks… too bad lint doesn't complain about this :)